<launch>
  <!-- 启动规划节点 -->
  <node name="astar_planner" pkg="astar_planner" type="planner_node" output="screen">
    <param name="start_x" value="0.0" />
    <param name="start_y" value="0.0" />
    <param name="start_z" value="0.5" />
    <param name="goal_x" value="5.0" />
    <param name="goal_y" value="5.0" />
    <param name="goal_z" value="0.5" />
  </node>

  <!-- 启动RViz -->
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find astar_planner)/config/planner.rviz" />
</launch>